src/components/Landing/QAs.tsx

```diff
@@ -19,20 +19,32 @@ export default function QAs() {
     {
       q: t`What does Juicebox cost?`,
       a: [
-        t`Juicebox is an open protocol on Ethereum that is funded using Juicebox itself. You can check out the contractualized budget specs at https://juicebox.money/#/p/juicebox.`,
+        <Trans>
+          Juicebox is an open protocol on Ethereum that is funded using Juicebox
+          itself. You can check out the contractualized budget specs{' '}
+          <a
+            href="https://juicebox.money/#/p/juicebox"
+            rel="noreferrer"
+            target="_blank"
+          >
+            {' '}
+            here
+          </a>
+          .
+        </Trans>,
         t`5% of all money made by projects is sent to help pay for the maintenance and development of Juicebox itself. In exchange, projects get Juicebox tokens ($JBX), which will be worth more as the ecosystem grows over time.`,
```

Comment:
Maybe here:


Projects building on Juicebox pay a 5% JBX membership fee from withdrawn funds into the JuiceboxDAO treasury. Projects can then use their JBX to help govern how the total JuiceboxDAO treasury is used. This fee is also subject to change via JBX member votes.

Reply:
Yeah love it. Maybe we could also add:

Projects can then use their JBX to help govern how the overall JuiceboxDAO treasury is used, as well as redeem from its overflow. 

Wdyt?